docs/framework/social-post-generator.mdx

```diff
@@ -2,218 +2,410 @@
 title: "Social post generator"
```

Comment:
I've run out of time to go through this one step by step, but you could apply basically all of the suggestions for the chat assistant one to this one (e.g. linking to guides, etc) and then tag @ajot to step through the social post one.

Reply:
👍